src/components/screens/userScreen/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from "react";
+import "./UserScreen.css";
+import { navigate } from "@reach/router";
+import { useUser } from "../../../providers/UsersProvider";
+
+const UserScreen = () => {
+  const provider = useUser();
+
+  useEffect(() => {
```

Comment:
Umesto globalnog efekta koji "imperativno" redirektuje korisnika, uraditi to "deklarativno".

[Imperativno vs deklarativno](https://codeburst.io/declarative-vs-imperative-programming-a8a7c93d9ad2)
[Primer](https://github.com/kentcdodds/bookshelf/blob/master/src/app.js)